django/core/handlers/base.py

```diff
@@ -167,7 +167,7 @@ def process_exception_by_middleware(self, exception, request):
             response = middleware_method(request, exception)
             if response:
                 return response
-        raise
+        raise response
```

Comment:
Your IDE is getting it wrong. `process_exception_by_middleware` is expected to be called within a an `except` block.

Reply:
thanks, you're right i just pip uninstall & reinstalled and pycharm is good now